snuba/datasets/configuration/events_analytics_platform/storages/eap_items_span.yaml

```diff
@@ -0,0 +1,253 @@
+version: v1
```

Comment:
add a comment that this is just a placeholder storage so it can be written to by the EAPItemsSpan consumer and should be removed